### PR TITLE
Add everness mineral water as dynamic liquid

### PR DESCRIPTION
**Pull request by BramvdnHeuvel**
The [everness](https://content.minetest.net/packages/SaKeL/everness/) mod contains a mineral water liquid. Sadly, it does not follow dynamic liquid rules.

This pull request aims to categorize mineral water as water. Players who install both the `dynamic_liquid` and `everness` mod, will see mineral water behave just like ordinary water.

One potential argument for denial of this pull request, is the argument that _that_ mod should contain cross-compatibility with this mod, not the other way around.

### CODE REVIEW

**Review by Niklp09 — APPROVED** [review]
LGTM, I asked the author of everness if they plans to add dynamic_liquid support on the things side of things, I'm still waiting for an answer.

**Comment by SmallJoker** [comment]
@Niklp09 Is there yet any response? If not - let's say we merge this PR on May 5 (if not forgotten) to provide compatibility from this mod and not in reverse.

Side note: The `everness` soft-dependency is not strictly necessary but still nice to see in the world's mod selector menu.

**Comment by Niklp09** [comment]
No response yet, btw I already had the 5th May as a good merge day on my list:).

**Comment by Niklp09** [comment]
Thank you for contributing @BramvdnHeuvel

**Comment by JurajVajda** [comment]
Hi, sorry to be late to the party, I was not aware of this mod and no open issues/PRs were found in the Everness repo. Nevertheless I can have a look on implementing also on our side.

**Pull request merged**
